src/node_options.cc

```diff
@@ -299,6 +306,10 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "show stack traces on process warnings",
             &EnvironmentOptions::trace_warnings,
             kAllowedInEnvironment);
+  AddOption("--unhandled-rejections",
+            "define unhandled rejections behavior",
```

Comment:
nit: May be its good to add here, the possible options? `strict`, `warn`, `none`.. Like `define unhandled rejections behavior. Options are `strict`, `warn`, `none``.